.azure-pipelines/build-template.yml

```diff
@@ -41,6 +41,10 @@ jobs:
 - job:
   displayName: ${{ parameters.arch }}
   timeoutInMinutes: ${{ parameters.timeout }}
+  variables:
+    DIFF_COVER_CHECK_THRESHOLD: 0
+    ${{ if ne(parameters.run_unit_test, false) }}:
+      DIFF_COVER_DISABLE: 'false'
```

Comment:
>DIFF_COVER_DISABLE [](http://example.com/codeflow?start=6&length=18)

Negative name is harder to read than postiive one. Suggest `DIFF_COVER_ENABLED` with default value `true`. Not sure how difficult for you to change code and redeploy.

Reply:
Maybe some day we can change the default value to false, so no one need to set the value. The feature is in preview status, we do not want to have any impact, so disable it by default.

Reply:
I am not asking to change the semantics of default value, only the name. The negative default value is actually same as your intention.

Reply:
Fixed, change to use DIFF_COVER_ENABLE=true.